core/test/net/sf/openrocket/aerodynamics/BarrowmanCalculatorTest.java

```diff
@@ -231,4 +231,50 @@ public void testRadialDiscontinuityWithStrapOns() {
 		
 		assertFalse(" Missed discontinuity in Falcon 9 Heavy:", calc.isContinuous( rocket));
 	}
+
+	@Test
+	public void testPhantomTubes() {
+		Rocket rocketNoPods = TestRockets.make3FNCNoPods();
+		FlightConfiguration configNoPods = rocketNoPods.getSelectedConfiguration();
+		FlightConditions conditionsNoPods = new FlightConditions(configNoPods);
+		WarningSet warningsNoPods = new WarningSet();
+		
+		Rocket rocketWithPods = TestRockets.make3FNCWithPods();
+		FlightConfiguration configPods = rocketWithPods.getSelectedConfiguration();
+		FlightConditions conditionsPods = new FlightConditions(configPods);
+		WarningSet warningsPods = new WarningSet();
+		AerodynamicCalculator calcPods = new BarrowmanCalculator();
+		AerodynamicCalculator calcNoPods = new BarrowmanCalculator();
+
+		final AerodynamicForces forcesNoPods = calcPods.getAerodynamicForces(configNoPods, conditionsNoPods, warningsNoPods);
+		final AerodynamicForces forcesPods = calcPods.getAerodynamicForces(configPods, conditionsPods, warningsPods);
+		assertEquals(" 3FNC With Pods rocket CD value is incorrect:", forcesPods.getCD(), forcesNoPods.getCD(), EPSILON);
+
+		// The "with pods" version has no way of seeing the fins are
+		// on the actual body tube rather than the phantom tubes,
+		// so CD won't take fin-body interference into consideration.
+		// So we'll adjust our CD in these tests.  The magic numbers
+		// in x and w come from temporarily disabling the
+		// interference calculation in FinSetCalc and comparing
+		// results with and without it
+		// cpNoPods (0.34125,0.00000,0.00000,w=16.20502) -- interference disabled
+		// cpNoPods (0.34797,0.00000,0.00000,w=19.34773) -- interference enabled
+
+		// another note:  the fact that this is seen as three one-fin
+		// FinSets instead of a single three-fin FinSet means the CP
+		// will be off-axis (one of the fins is taken as having an
+		// angle of 0 to the airstream, so it has no contribution).
+		// This doesn't turn out to cause a problem in an actual
+		// simulation, so we are just not testing for it.  Test with
+		// correction if we want some time is here but commented out
+		
+		final Coordinate cpNoPods = calcNoPods.getCP(configNoPods, conditionsNoPods, warningsNoPods);
+		final Coordinate cpPods = calcPods.getCP(configPods, conditionsPods, warningsPods);
+		
+		assertEquals(" 3FNC With Pods rocket cp x value is incorrect:", cpPods.x, cpNoPods.x - 0.00672, EPSILON);
+		// assertEquals(" 3FNC With Pods rocket cp y value is incorrect:", cpPods.y, cpNoPods.y - 0.00548, EPSILON);
+		// assertEquals(" 3FNC With Pods rocket cp z value is incorrect:", cpPods.z, cpNoPods.z, EPSILON);
```

Comment:
are these commented out for a reason?

Reply:
Yes.  When the center of pressure is calculated for a with an odd number of fins, there is a Y component to the calculated CP because one of the fins is taken as being at angle of 0 and doesn't contribute to the CP; the error doesn't end up affecting flight simulations because the rocket is assumed to only vary in pitch (which strikes me as something to fix to make the simulation more accurate, but that's for another day), so indeed that fin doesn't contribute.
When you've got  a finset with three or more fins, the Y and Z components of the CP are forced to 0; that's not practical when you're putting together the CP from fins from different instances.
My initial thought was to explicitly run the test with a correction for the offset, but upon reflection it seemed like writing an error into the test case was a mistake.  So, I commented out the Y and Z tests, and added the comment just before the tests.

Reply:
@JoePfeiffer 
> When the center of pressure is calculated for a with an odd number of fins, there is a Y component to the calculated CP because one of the fins is taken as being at angle of 0 and doesn't contribute to the CP; the error doesn't end up affecting flight simulations because the rocket is assumed to only vary in pitch (which strikes me as something to fix to make the simulation more accurate, . . .

Yea, I ran into this in relation to https://github.com/openrocket/openrocket/issues/538
And may be fixing that.  Let me know in a separate thread if you have opinions on the right way to do so.